src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
Ok, thanks for clarifying. But builtins is part of the standard library; if we rewrite the line(s) where listitems is used (it should be easy), can we completely drop the future dependency? Sorry, I'm a bit confused.

Should we drop the global analysis now?